### DISCUSSION TITLE
V 3.06 - no workspace / delivery settings tab

### DISCUSSION THREAD

**Discussion by Stefan-JustMe69-SQ** [BrandOriented/PimcoreCiHubConnector#17] [Q&A] [answered]
Hi,
I'm using pimcore 11 and actual versions of [PimcoreCiHubConnector](https://github.com/BrandOriented/PimcoreCiHubConnector) and [PimcoreCIHubAdapterBundle](https://github.com/BrandOriented/PimcoreCIHubAdapterBundle)

After updating to V 3.0.6 (Connector) there is no tab in pimcore backend anymore on CI-hub configuration 'workspace' and 'delivery settings' as shown in your pictures.
So I'm not able to define api-key and connot use CI-Hub as I cannot define the token for usage?

Please help/advise

kind regards

**Comment by labudzinski** [comment]
https://github.com/BrandOriented/PimcoreCiHubConnector/issues/18

**Comment by dccfena** [comment]
Hi,
we are also struggling with this new way of the configuration.

As per default the lines 83-108 in EndpointAndIndexesConfigurator.php are triggering an indexing of ALL Assets in our DAM - the workspaces are not taken into account during execution of the index. With around 130k assets this is crashing the server - even when we only want to use CI Hub for a small part of them.

Can you suggest a workaround for this issue?

**Reply by labudzinski** [reply, marked as answer]
@dccfena #28 please confirm this solves the problem.
